content/docs/command-reference/exp/save.md

```diff
@@ -33,8 +33,7 @@ possible to quickly start tracking, [comparing] and [persisting] experiments.
 ## Options
 
 - `-n <name>`, `--name <name>` - specify a [unique name] for this experiment. A
-  default one will be generated otherwise, such as `exp-f80g4` (based on the
-  experiment's hash).
+  default one will be generated otherwise, such as `urban-sign`.
```

Comment:
```suggestion
  default one will be generated otherwise, such as `urban-sign`.

  <admon type="tip">

  The name of the experiment is exposed in env var `DVC_EXP_NAME`.

  </admon>
```